server/services/premisesService.ts

```diff
@@ -80,6 +80,13 @@ export default class PremisesService {
       })
   }
 
+  async getPremises(token: string, id: string): Promise<Premises> {
+    const premisesClient = this.premisesClientFactory(token)
+    const premises = (await premisesClient.find(id)) as Premises
```

Comment:
Minor, but do we need to cast this? I would've thought the return type could be inferred from the `find` method

Reply:
Ah, yep, we don't, it's already coming from the client with the correct type. Thanks!